bitnami/mongodb/templates/replicaset/scripts-configmap.yaml

```diff
@@ -137,4 +152,102 @@ data:
     export MONGODB_EXTRA_DATABASES_FILE=""
     export MONGODB_EXTRA_PASSWORDS_FILE=""
     exec /opt/bitnami/scripts/mongodb/entrypoint.sh /opt/bitnami/scripts/mongodb/run.sh
+  {{- if .Values.replicaSetConfigurationSettings.enabled }}
+  replicaSetConfigurationSettings.sh: |-
+    #!/bin/bash
+    # This script to be called when pod starts.
+    # This script sets rs settings which can not be applied via conf file
+
+    function logger ()
+    #$1 is the line to be logged
```

Comment:
Instead of creating this logger function you could add . /opt/bitnami/scripts/liblog.sh and use the functions from there. You can take look to https://github.com/bitnami/bitnami-docker-mongodb/blob/master/4.4/debian-10/rootfs/opt/bitnami/scripts/libmongodb.sh for some examples of the usage.
There are also functions for warning and error messages.